### PR TITLE
add rune magic and descriptions

### PR DESCRIPTION
**Pull request by TheChaosium**
rune magic spells split out and added to the system

### CODE REVIEW

File: src/assets/pack-templates/rune-magic-spells/absorption.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  isStackable: true
+  isOneUse: false
+  chance: 0
+  descriptionRqidLink:
+    rqid: je..absorption
+    name: ${{openSystem.content.runeMagic.absorption.name}}$
+flags:
+  rqg:
+    documentRqidFlags:
+      id: i.runeMagic.absorption
```
Comment (wake42) [inline src/assets/pack-templates/rune-magic-spells/absorption.yaml:21]: rqid shouldn't be camel case. Goes for the other runemagic rqids as well.
```suggestion
      id: i.rune-magic.absorption
```

File: src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  isStackable: true
+  isOneUse: false
+  chance: 0
+  descriptionRqidLink:
+    rqid: je..analyze-magic
+    name: ${{openSystem.content.runeMagic.analyzeMagic.name}}$
+flags:
+  rqg:
+    documentRqidFlags:
+      id: i.runeMagic.rune-magic-analyze-magic
```
Comment (wake42) [inline src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml:20]: Double "rune magic" in id, applies to more rqids as well. 
```suggestion
      id: i.rune-magic.analyze-magic
```

**Review by wake42 — CHANGES_REQUESTED** [review]

File: src/assets/pack-templates/rune-magic-spells/absorption.yaml (hunk 1 above)
Comment (TheChaosium) [reply src/assets/pack-templates/rune-magic-spells/absorption.yaml:21]: Easy fix. I did't change it from the starter set version though so we missed it before. But for tomorrow.

File: src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml (hunk 2 above)
Comment (TheChaosium) [reply src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml:20]: Again, I just did file movement so this we missed before. I will review all spells and descriptions. The description links seems to work so it is most likely just the spells (and we don't link to them yet)

Comment (TheChaosium) [reply src/assets/pack-templates/rune-magic-spells/analyze-magic.yaml]: Cleaned this up and removed links to other spells that will not work.

**Review by wake42 — APPROVED** [review]

**Pull request merged**